torch/_refs/__init__.py

```diff
@@ -4687,21 +4687,25 @@ def scalar_tensor(
 
 @register_decomposition(torch.ops.aten.uniform)
 def uniform(
-    shape: ShapeType,
+    x: TensorLikeType,
```

Comment:
Not sure if x should be tensor or shape, as ```aten::uniform()``` expects tensor as the first argument:
```
>>> import torch
>>> torch.ops.aten.uniform([2, 3])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/scratch/ybliang/work/repos/pytorch/torch/_ops.py", line 500, in __call__
    return self._op(*args, **kwargs or {})
RuntimeError: Overloaded torch operator invoked from Python failed to many any schema:
aten::uniform() Expected a value of type 'Tensor' for argument 'self' but instead found type 'list'.
Position: 0
Value: [2, 3]
Declaration: aten::uniform(Tensor self, float from=0., float to=1., *, Generator? generator=None) -> Tensor
Cast error details: Unable to cast [2, 3] to Tensor

aten::uniform() Expected a value of type 'Tensor' for argument 'self' but instead found type 'list'.
Position: 0
Value: [2, 3]
Declaration: aten::uniform.out(Tensor self, float from=0., float to=1., *, Generator? generator=None, Tensor(a!) out) -> Tensor(a!)
Cast error details: Unable to cast [2, 3] to Tensor
```

Reply:
And dispatch is at https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L7734